src/pykeen/nn/functional.py

```diff
@@ -1294,3 +1296,55 @@ def cp_interaction(
         r,
         t,
     )
+
+
+def transformer_interaction(
+    h: torch.FloatTensor,
+    r: torch.FloatTensor,
+    t: torch.FloatTensor,
+    transformer: nn.TransformerEncoder,
+    position_embeddings: torch.FloatTensor,
+    final: nn.Module,
+) -> torch.FloatTensor:
+    """Evaluate the Transformer interaction function, as described in [galkin2020]_..
+
```

Comment:
Please add a beautiful math equation in latex

Reply:
9a4c45e

Reply:
Thank you :)